app/components/security_key_image_component.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class SecurityKeyImageComponent < BaseComponent
+  def initialize(mobile:)
+    @mobile = mobile
+  end
+
+  def mobile?
+    !!@mobile
+  end
+
+  def svg_tag
+    # rubocop:disable Rails/OutputSafety
+    @svg_tag ||= Nokogiri::HTML5.fragment(read_svg).tap do |doc|
+      doc.at_css('svg').tap do |svg|
+        svg[:height] = 193
+        svg[:width] = 420
+        svg[:class] = css_class
+        svg[:role] = 'img'
+
+        svg << "<title>#{title}</title>"
+      end
+    end.to_s.html_safe
```

Comment:
unfortunately, Nokogiri's documents are not considered HTML-safe by Rails (even though by most accounts, they should be)